### PR TITLE
Added board view and delete task functionality

### LINKED ISSUE #16: Default board view and delete task
Board view: Basic UI using bootstrap framework consisting of a single "TO DO" form

Delete task: Button to delete tasks from task list from a given user input. Tasks are referred by their taskID

### PR DESCRIPTION
**Pull request by AaronDzf**
• Board view redirected from login page
• Includes button to expand ListView

• Delete task removes task by taskID (not task number), does not display any message for invalid user input into the user form

### CODE REVIEW

File: boardView.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          <a class="dropdown-item" href="#">Something else here</a>
+        </div>
+      </li>
+    </ul>
+    <form class="form-inline my-2 my-lg-0">
+      <input class="form-control mr-sm-2" type="text" placeholder="Search" aria-label="Search">
+      <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+    </form>
+  </div>
+</nav>
```
Comment (Rolo123y) [inline boardView.html]:
```suggestion
    <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
  <a class="navbar-brand" href="#">Student Planner</a>
  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
    <span class="navbar-toggler-icon"></span>
  </button>

  <div class="collapse navbar-collapse" id="navbarsExampleDefault">
    <ul class="navbar-nav mr-auto">
      <li class="nav-item active">
        <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#">Link</a>
      </li>
      <li class="nav-item">
        <a class="nav-link disabled" href="#" tabindex="-1" aria-disabled="true">Disabled</a>
      </li>
      <li class="nav-item dropdown">
        <a class="nav-link dropdown-toggle" href="#" id="dropdown01" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Dropdown</a>
        <div class="dropdown-menu" aria-labelledby="dropdown01">
          <a class="dropdown-item" href="#">Action</a>
          <a class="dropdown-item" href="#">Another action</a>
          <a class="dropdown-item" href="#">Something else here</a>
        </div>
      </li>
    </ul>
    <form class="form-inline my-2 my-lg-0">
      <input class="form-control mr-sm-2" type="text" placeholder="Search" aria-label="Search">
      <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
    </form>
  </div>
</nav>
```
This is not needed and can be removed!

**Review by Rolo123y — CHANGES_REQUESTED** [review]
**boardView.html:**
- The UI for the BoardView will need tweaking later on but is a good start!

**deleteTask.php:**
- Can look into integrating this into html later on to output a neater looking list

**Review by Rolo123y — APPROVED** [review]

**Review by Rolo123y — COMMENTED** [review]
the nav is now removed. Looks good!

**Comment by gaurav613** [comment]
This looks good! We can try to integrate list(s) within the board in the future, rather than having an expand button.

**Pull request merged**